go.mod

```diff
@@ -6,32 +6,43 @@ require (
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.61.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.61.0
-	github.com/prometheus/common v0.37.0
+	github.com/jaegertracing/jaeger v1.41.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.70.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.70.0
+	github.com/prometheus/common v0.39.0
 	github.com/stretchr/testify v1.8.1
-	go.opencensus.io v0.23.0
-	go.opentelemetry.io/collector v0.61.0
-	go.opentelemetry.io/collector/pdata v0.61.0
-	go.opentelemetry.io/collector/semconv v0.61.0
-	go.opentelemetry.io/otel/trace v1.10.0
-	go.uber.org/multierr v1.8.0
-	go.uber.org/zap v1.23.0
-	google.golang.org/grpc v1.49.0
+	go.opencensus.io v0.24.0
+	go.opentelemetry.io/collector v0.70.0
+	go.opentelemetry.io/collector/component v0.70.0
+	go.opentelemetry.io/collector/consumer v0.70.0
+	go.opentelemetry.io/collector/exporter/loggingexporter v0.70.0
+	go.opentelemetry.io/collector/exporter/otlpexporter v0.70.0
+	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.70.0
+	go.opentelemetry.io/collector/extension/ballastextension v0.70.0
+	go.opentelemetry.io/collector/extension/zpagesextension v0.70.0
+	go.opentelemetry.io/collector/pdata v1.0.0-rc4
```

Comment:
oh this one is different? In know that for some packages they have v1.0.0-rc4 and others they have v0.70.0

Reply:
I noticed that at some point also.  I did not put it to that value.  I used `v0.70.0`.  But `go mod tidy` will change it to the `v1.0.0-rc4`. (both those versions are related in the opentelemetry collector repo)